src/OpenLoco/src/Vehicles/Vehicle.h

```diff
@@ -460,15 +448,20 @@ namespace OpenLoco::Vehicles
     {
         static constexpr auto kVehicleThingType = VehicleEntityType::vehicle_1;
         uint8_t pad_24[0x26 - 0x24];
-        EntityId head;                       // 0x26
-        uint32_t remainingDistance;          // 0x28
-        TrackAndDirection trackAndDirection; // 0x2C
-        uint16_t subPosition;                // 0x2E
-        int16_t tileX;                       // 0x30
-        int16_t tileY;                       // 0x32
-        World::SmallZ tileBaseZ;             // 0x34
-        uint8_t trackType;                   // 0x35 field same in all vehicles
-        RoutingHandle routingHandle;         // 0x36 field same in all vehicles
+        EntityId head;              // 0x26
+        uint32_t remainingDistance; // 0x28
+        union
+        {
+            TrackAndDirection trackAndDirection;
+            RoadAndDirection roadAndDirection;
+            uint16_t trad;
```

Comment:
what is this for? `trad` isn't a very descriptive name

Reply:
track or road and direction is a mouthful. Its there for the cases where we don't know what its going to be used for.